snapshots/health/2025-02-20/measles_deaths_historical.csv.dvc

```diff
@@ -0,0 +1,63 @@
+
+      * 1938: Public Health Reports, Vol. 55, No. 10 - [Notifiable Diseases in the United States, 1938: Morbidity and Mortality Summaries for Certain Important Communicable Diseases](https://www.jstor.org/stable/4583204)
+
+      * 1939: Public Health Reports, Vol. 56, No. 7 - [Notifiable Diseases in the United States, 1939: Morbidity and Mortality Summaries for Certain Important Communicable Diseases](https://www.jstor.org/stable/4583620)
+
+      * 1940: Public Health Reports, Vol. 57, No. 7 - [Notifiable Diseases in the United States, 1940: Morbidity and Mortality Summaries for Certain Important Communicable Diseases](https://www.jstor.org/stable/4584013)
+
+      * 1949: Statistical Abstract of the United States: 1952, United States Census Bureau - [Part 2 - Vital Statistics and Health and Medical Care](https://www.census.gov/library/publications/1952/compendia/statab/73ed.html)
+
+    date_published: "1952-09-01"
+
+    # Citation
+    producer: Public Health Reports; US Census Bureau
+    citation_full: |-
+      Public Health Reports, Vol. 36, No. 8. The Notifiable Diseases: Prevalence during 1919 in States.
+
+      Public Health Reports Vol. 37, No. 41. The Notifiable Diseases: Prevalence during 1921 in States. Anthrax, Cerebrospinal Meningitis, Dengue, Diphtheria, Gonorrhea, Influenza, Malaria, Measles, Pneumonia, Poliomyelitis, Rabies, Rocky Mountain Spotted Fever, Scarlet Fever, Septic Sore Throat, Smallpox, Syphilis, Tuberculosis (All Forms and Pulmonary), Typhoid Fever, and Typhus Fever.
+
+      Public Health Reports Vol. 40, No. 51. The Notifiable Diseases: Prevalence during 1924 in States.
+
+      Public Health Reports, Vol. 42, No. 1 - The Notifiable Diseases: Prevalence during 1925 in States.
+
+      Public Health Reports, Vol. 53, No. 44 - Health Conditions in the United States as Indicated by Mortality Rates, 1929-38.
+
+      Public Health Reports, Vol. 55, No. 10 - Notifiable Diseases in the United States, 1938: Morbidity and Mortality Summaries for Certain Important Communicable Diseases.
+
+      Public Health Reports, Vol. 56, No. 7 - Notifiable Diseases in the United States, 1939: Morbidity and Mortality Summaries for Certain Important Communicable Diseases.
+
+      Public Health Reports, Vol. 57, No. 7 - Notifiable Diseases in the United States, 1940: Morbidity and Mortality Summaries for Certain Important Communicable Diseases.
+
+      US Census Bureau. Part 2 - Vital Statistics, Statistical Abstract of the United States 1944-45.
+
+    # Files
+    url_main: https://www.jstor.org/stable/4575902; https://www.jstor.org/stable/4576538; https://www.jstor.org/stable/4577735; https://www.jstor.org/stable/4578131; https://stacks.cdc.gov/view/cdc/69651/cdc_69651_DS1.pdf; https://www.jstor.org/stable/4583204; https://www.jstor.org/stable/4583620; https://www.jstor.org/stable/4584013; https://www.census.gov/library/publications/1952/compendia/statab/73ed.html
```

Comment:
Hmm, I think this could be problematic, `url_main` is expected to be just one URL. Let me see how this works out in the end...